06_correlation_analysis.ipynb

```diff
@@ -0,0 +1,1984 @@
+{
```

Comment:
<p>I guess you are only looking at this gene here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/pancancer-evaluation/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>For now, yeah, but I'd like to have the option to run it for other genes in the future.</p><p> </p><p>How do you usually handle arguments/parameters to notebooks, and running them multiple times for different parameters? I think I remember that Jupytext has some options for this - is that what you use? </p>

Reply:
Ahh, nice! I'll give this a try in the future.